collector/usage.go

```diff
@@ -64,13 +69,18 @@ func (c UsageCollector) collect(ch chan<- prometheus.Metric) error {
 			ch <- prometheus.MustNewConstMetric(
 				c.Quota,
 				prometheus.GaugeValue,
-				float64(data.Quota),
+				float64(actualUserQuota),
 				"user", data.User.Name, strconv.FormatInt(int64(data.User.Id), 10), data.FileSystem["name"])
 		}
 	}
 
 	for _, usage := range groupUsage {
 		for _, data := range usage.Items {
+            actualGroupQuota := data.Quota
+            if actualGroupQuota == 0 {
+                actualGroupQuota = data.FileSystemDefaultQuota
```

Comment:
Since the logic is the same in both places, I'd pull this into a function with a comment explaining that the FlashBlade API shows 0 for both user and group quotas if an explicit quota isn't set. What do you think?